servers/rendering/dummy/rasterizer_canvas_dummy.h

```diff
@@ -52,7 +52,7 @@ class RasterizerCanvasDummy : public RendererCanvasRender {
 	void occluder_polygon_set_cull_mode(RID p_occluder, RS::CanvasOccluderPolygonCullMode p_mode) override {}
 	void set_shadow_texture_size(int p_size) override {}
 
-	bool free(RID p_rid) override { return true; }
+	bool free(RID p_rid) override { return false; }
```

Comment:
Changed this as it otherwise prevents calling `free` in other cases, see:
https://github.com/godotengine/godot/blob/fa48a51183567934984b381ad8ec281cb24d66ba/servers/rendering/rendering_server_default.cpp#L47-L63

Reply:
Realized this was a layer further down and wasn't used in this way, removed